src/Person/PersonProvider.php

```diff
@@ -20,18 +24,356 @@ public function __construct(Client $client)
         $this->client = $client;
     }
 
+    public function getConfig(): ?ConfigDto
+    {
+        try {
+            $response = $this->client->get('/api/persons/config', [
+                "headers" => [
+                  "Content-Type" => "application/json",
+                ]
+          ]);
+
+            if ($response->getStatusCode() >= 300) {
+                throw new SherlException(PersonProvider::DOMAIN, $response->getBody()->getContents(), $response->getStatusCode());
+            }
+
+            return SerializerFactory::getInstance()->deserialize(
+                $response->getBody()->getContents(),
+                ConfigDto::class,
+                'json'
+            );
+        } catch (\Exception $e) {
+            throw new SherlException(SherlException::FETCH_FAILED, $e->getMessage());
+        }
+
+    }
+
+    /**
+    * @return Pagination<LocationDto>|null
+    */
+    public function getCurrentAddress(GeoCoordinatesDto $position)
+    {
+        try {
+            $response = $this->client->get('/api/persons/current-address', [
+                "headers" => [
+                  "Content-Type" => "application/json",
+                ],
+                RequestOptions::QUERY => $position
+          ]);
+
+            if ($response->getStatusCode() >= 300) {
+                throw new SherlException(PersonProvider::DOMAIN, $response->getBody()->getContents(), $response->getStatusCode());
+            }
+
+            return SerializerFactory::getInstance()->deserialize(
+                $response->getBody()->getContents(),
+                LocationDto::class,
+                'json'
+            );
+        } catch (\Exception $e) {
+            throw new SherlException(SherlException::FETCH_FAILED, $e->getMessage());
+        }
+
+    }
+
+    /**
+    * @return Pagination<PersonOuputDto>|null
+    */
+    public function getPersons(
+        $page = 1,
+        $itemsPerPage = 10,
+        PersonFiltersDto $filters
+    ) {
+        try {
+            $response = $fetcher->get(
```

Comment:
Les appels à l'API sont fait différemment que sur le SDK JS. En PHP on ne va pas utiliser de `$fetcher` (au passage cette ligne ne retourne-t-elle pas une erreur dans l'IDE ?) mais plutôt un client Guzzle, appelable avec `$this->client`.

```suggestion
            $response = $this->client->get(
```

À répercuter sur les autres routes qui utilisent `$fetcher`.